Add buggy test for PhpdocSeparationFixer

keradus opened the pull request:
Don't know why but PhpdocSeparationFixer stick `@param` with `@access`.

Could you take care of this @GrahamCampbell ?

GrahamCampbell commented:
That seemed to be the preferred thing people did when I looked through a crap ton of php repos.

keradus commented:
We split `@param` from `@return`. I can see some sense of keeping them together, but we split it. Next, I see no sense to keep `@param` with `@access`.

> Annotations in phpdocs should be grouped together so that annotations of the same type immediately follow each other, and annotations of a different type are separated by a single blank line.

They are not the same type, didn't they?


Great thing about this Fixer is when sb used crappy style on his docs then this fixer... fix it ;)

GrahamCampbell commented:
I'll look into this at the weekend. Ping me if I don't do anything by Saturday evening. :)

keradus commented:
Thank you!

(I won't ping you, I will be offline this weekend)

keradus commented:
ping @GrahamCampbell

GrahamCampbell commented:
People shouldn't even be using the access annotation - it's not even a thing. I propose we add a fixer to remove such annotations like we have for the package/subpackage annotations.

keradus commented:
very well.

GrahamCampbell commented:
Can this be closed then?

keradus commented:
Closing due to merge of #1144
